dabl-meta.yaml

```diff
@@ -4,13 +4,14 @@
 catalog:
     name: dabl-integration-core
     group_id: com.digitalasset
-    version: 0.5.2
-    short_description: Core Integrations
+    version: 0.5.4
+    short_description: The Core Pack
     description: Timer and Loopback Integrations
-    release_date: 2020-09-19
+    release_date: 2020-09-30
```

Comment:
Today is actually October 1, believe it or not… :-)

Reply:
This is why `release_date` is the first deprecated metadata field. (And no longer displayed on the UI.) :-)